### PR TITLE
Add fetch extension to API response

### PR DESCRIPTION
**Pull request by dhardiman**

### CODE REVIEW

**Comment by dhardiman** [comment]
Also added standard set of error responses based on the property API documentation.

**Comment by dhardiman** [comment]
Bump for slack notification

File: OSAPIResponse/ResponseErrors.swift [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+//  OSAPIResponse
+//
+//  Created by Dave Hardiman on 15/03/2016.
+//  Copyright © 2016 Ordnance Survey. All rights reserved.
+//
+
+import Foundation
+
+public enum ResponseErrors: ErrorType {
+    case UknownError
```
Comment (drhaynes) [inline OSAPIResponse/ResponseErrors.swift]: `Unknown`

Comment (dhardiman) [inline OSAPIResponse/ResponseErrors.swift]: Already fixed in the rest of the commits...

Comment (drhaynes) [inline OSAPIResponse/ResponseErrors.swift]: ah yeah

**Comment by drhaynes** [comment]
:+1: Will be good to get this into Search

**Pull request merged**